fastly/fixtures/wasm_package/get.yaml

```diff
@@ -0,0 +1,48 @@
+---
+version: 1
+interactions:
+- request:
+    body: ""
+    form: {}
+    headers:
+      User-Agent:
+      - FastlyGo/1.15.0 (+github.com/fastly/go-fastly; go1.14.3)
+    url: https://api.fastly.com/service/2Q0B1WlDbii9DItrMCvIGL/version/2/package
+    method: GET
+  response:
+    body: '{"version":2,"deleted_at":null,"updated_at":"2020-06-23T11:11:37Z","created_at":"2020-06-23T11:11:34Z","id":"0eh4MAGwaPjZerrjj44ZI6","metadata":{"name":"wasm-test","description":"Default
+      package template used by the Fastly CLI for Rust-based Compute@Edge projects.","authors":["guy.richardson+fastly@opencredo.com"],"language":"rust","size":2179443,"hashsum":"119a5ac6ec8a6dc29107b87de3bba5d98b6e89827310dd01f4e8cbf89234125bf4f7b81603ac7df42ce0205d047ac2caa53ec0cc52839112aed786cb7f27ac92"},"service_id":"2Q0B1WlDbii9DItrMCvIGL"}'
```

Comment:
Not sure if you want your email in there. You could replace with some fake address.

Reply:
Have removed from the metadata